homeassistant/components/sql/config_flow.py

```diff
@@ -176,7 +179,7 @@ async def async_step_init(
             step_id="init",
             data_schema=vol.Schema(
                 {
-                    vol.Required(
+                    vol.Optional(
```

Comment:
Ideally we have a test for this

Reply:
Closed down vscode for the night so I can write one up tomorrow and add it in before we merge it.